app/assets/scss/partials/_app.scss

```diff
@@ -22,34 +21,35 @@
 }
 
 .app-component-preview {
-  position: relative;
```

Comment:
Can you explain the reasoning behind removing this property please?

Reply:
I wasn't supposed to commit this until a further merge request, but the immediate child `.app-component-preview__iframe` already has `position: relative`.

It doesn't appear necessary to have it on both.

I've put it back, it's not the right commit to remove lines—only lint them.